main.nf

```diff
@@ -1236,28 +1244,36 @@ if (params.binning_map_mode == 'all'){
 
 process bowtie2 {
     tag "$assembler-$name"
+    publishDir "${params.outdir}/Assembly/${assembler}/${name}_QC", mode: params.publish_dir_mode,
```

Comment:
this is not added to the `output.md` yet, if I see it right

Reply:
Done